src/test/java/com/google/api/generator/gapic/composer/ServiceClientClassComposerTest.java

```diff
@@ -223,17 +223,16 @@ public void generateServiceClasses() {
           + "    return stub.chatAgainCallable();\n"
           + "  }\n"
           + "\n"
-          + "  public final PagedExpandResponse pagedExpand(PagedExpandRequest request) {\n"
-          + "    return pagedExpandCallable().call(request);\n"
+          + "  public final PagedExpandPagedResponse pagedExpand(PagedExpandRequest request) {\n"
+          + "    return pagedExpandPagedCallable().call(request);\n"
           + "  }\n"
           + "\n"
           + "  public final UnaryCallable<PagedExpandRequest, PagedExpandPagedResponse>\n"
           + "      pagedExpandPagedCallable() {\n"
           + "    return stub.pagedExpandPagedCallable();\n"
           + "  }\n"
           + "\n"
-          + "  public final UnaryCallable<PagedExpandRequest, PagedExpandResponse>"
-          + " pagedExpandCallable() {\n"
+          + "  public final UnaryCallable<PagedExpandRequest, PagedExpandResponse> pagedExpandCallable() {\n"
```

Comment:
This looks wired to me. 
Should it looks like
```
+ "  public final UnaryCallable<PagedExpandRequest, PagedExpandResponse>"
+ " pagedExpandCallable() {\n"```

Reply:
I think that's because of the formatting tool, without this change, the unit test fails. :(